docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -38,23 +36,25 @@ You need the following to get invited to TechPass and complete the onboarding:
   >
   > **When you are invited to TechPass**
   >- A TechPass account is provisioned for you and is in pending state.
-  >- We'll send the TechPass onboarding email to activate the account. This email is valid only for 30 days and will have your TechPass account or log in ID. 
-  >- Your TechPass log in ID's domain will be ``techpass.gov.sg```.
+  >- We'll send the TechPass onboarding email with your TechPass account or log in ID. 
```

Comment:
Is it because its active voice style? cause typically i think we would do something like "An onboarding email will be sent"

Reply:
Yes and further here the active voice did not make it complex so i opted this.

Reply:
My take is if we want to use passive, then all use passive and same if we want to use active voice. Which sentence structure are we using <Action>, <Effect> or  <Action>, <Cause>?

a) "To access services such as SGTS and GCC 2.0 resources through an Internet Device, you need to onboard that device to SEED."

vs

b) "You need to onboard that device to SEED, to access services such as SGTS and GCC 2.0 resources through an Internet Device"

According to Microsoft style (https://learn.microsoft.com/en-us/style-guide/grammar/verbs#active-and-passive-voice), we should keep it active as in the subject of the sentence performs the action. I thought b) aligns better though the sentence needs some cleaning up.

feel free to disregard if i am not making sense here

Reply:
yep, we use active voice as it is clearer.